### PR TITLE
chore(Github): update issue template re-direct user for jira

### PR DESCRIPTION
**Pull request by zdtsw**
## Description

ref: https://issues.redhat.com/browse/RHOAIENG-989

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 1]
```diff
@@ -1,17 +1,43 @@
...
-A clear and concise description of any alternative solutions or features you've considered.
+All previous issues from GitHub have been seamlessly migrated to the issue board, accompanied by a link directing users to the corresponding old issues for follow-up.
 
-**Additional context**
-Add any other context or screenshots about the feature request here.
+**How To**
+
+- Login Jira issue board https://issues.redhat.com
+- Click "Create" on top
+- Choose: "Pojrect 'Red Hat Openshift Data Science'"
```
Comment (AjayJagan) [inline .github/ISSUE_TEMPLATE/feature_request.md:18]:
```suggestion
- Choose: "Project 'Red Hat Openshift Data Science'"
```

**Comment by AjayJagan** [comment]
lgtm :)

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,24 +1,16 @@
 ---
-name: Bug report v1
-about: Create a report to help us improve v1 operator
+name: End supporting for v1
```
Comment (AjayJagan) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]:
```suggestion
name: End support for v1
```

**Review by AjayJagan — APPROVED** [review]

**Review by ajaypratap003 — APPROVED** [review]
lgtm

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Pull request merged**